src/operator/nn/pooling-inl.h

```diff
@@ -50,6 +50,7 @@ struct PoolingParam : public dmlc::Parameter<PoolingParam> {
   bool global_pool;
   bool cudnn_off;
   dmlc::optional<int> p_value;
+  dmlc::optional<bool> count_include_pad;
```

Comment:
doesn't need to be optional. Just set to the previous default

Reply:
We want to ensure forward compatibility here, if this is not an optional field, json file generated for the symbol will have an extra field, which we think may cause confusions for users of earlier versions.